### DISCUSSION TITLE
Uncensored Llama 3.1 405b support on open-webui

### DISCUSSION THREAD

**Discussion by yeehi** [open-webui/open-webui#4543] [Q&A]
When might we be able to run such a model locally using open-webui? 
Can we run the standard version of Llama3.1 405b until then?
Can the inferences we run be answered using RAG (Retrieval Augmented Generation)?

Which hosting providers would let us easily setup a pay-as-you-use llama3.1 RAG inferencing?

**Comment by GrayXu** [comment]
groq, siliconflow, ...
